.travis.yml

```diff
@@ -16,7 +16,7 @@ env:
 matrix:
   include:
     - php: 7.1
-      env: SYMFONY_VERSION=2.8.* COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
+      env: SYMFONY_VERSION=2.8.* JACKALOPE_VERSION='>= 1.3.1' COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
why do you need this? jackalope/jackalope is the base library used by both jackalope-doctrine-dbal and jackalope-jackrabbit. jackalope is installed in the bash script below. should we add a conflict with older versions?

Reply:
Look at [this Travis build](https://travis-ci.org/damienflament/DoctrinePHPCRBundle/builds/332038348) on my repository.

The [build job for lowest dependencies](https://travis-ci.org/damienflament/DoctrinePHPCRBundle/jobs/332038351) failed.

I didn't investigate more on it. I just noted that this job use _jackalope/jackalope_ 1.3.0 but the while the Symfony 2.8 job use the 1.3.4 version.

Maybe should we investigate more on this.

P.S: the bash script from _symfony-cmf/testing_ does not install anything. It just run the command for initializing the PHPCR repository.

Jackalope is required by the _symfony-cmf/testing_ package:

```
$ composer why jackalope/jackalope

jackalope/jackalope-doctrine-dbal  dev-master  requires  jackalope/jackalope (^1.3.4)
symfony-cmf/testing                dev-master  requires  jackalope/jackalope (^1.3)
```

```
$ composer why jackalope/jackalope-doctrine-dbal

symfony-cmf/testing  dev-master  requires  jackalope/jackalope-doctrine-dbal (^1.3)
```

Reply:
hm, all very indirect. can you please add a conflicts section in here that says `"jackalope/jackalope-doctrine-dbal": "< 1.3.1"`?

Reply:
As the bug is present in _jackalope/jackalope_, I added it to the conflicting packages instead of _jackalope-doctrine-dbal_.